### DISCUSSION TITLE
Inconsolata Nerd Font is two versions behind upstream

### DISCUSSION THREAD

**Discussion by ggustafsson** [ryanoasis/nerd-fonts#774] [General]
Hello!

I want to try out the Inconsolata Nerd Font but I noticed that it is two versions behind upstream. It would be really nice if someone could bump it up.

v.3.000 is the latest version and v.2.011 is used in this project.

https://github.com/googlefonts/Inconsolata

**Comment by Finii** [comment]
Inconsolata DZ was my main font until Cascadia came out ;-)

Whatever. There are three 'Inconsolata's in Nerd Fonts, maybe it is also time to clean up.

1. `InconsolataGo` -> Is 'up to date' (i.e. Dec. 2015 [source](https://levien.com/type/myfonts/inconsolata.html))
2. `Inconsolata` -> Is '2.012' according to embedded version information
3. `Inconsolata LGC` -> Is 1.3, probably this https://github.com/MihailJP/Inconsolata-LGC (from Apr. 2016)

Looking at Inconsolata 3.0.0,
* it has now more weights, from 'extra light' to 'black', 6 in total
* it has now several widths, from 'ultra condensed' to 'ultra expanded', 9 in total
* it has a always ligature version (2 font files)
* it includes some JetBrainsMono variants (??!)

I must have missed some variants in weight/width, because there are **72** font files (`ls Inconsolata-*ttf | wc -l`)...

For me that is far too much. Maybe just update the 2 currently available styles (regular & bold)? :grimacing:

What do you _miss_ from 2.012 that 3.0.0 has?

**Reply by Finii** [reply]
BTW it's not **two** versions behind, it is just the last before current..:
![image](https://user-images.githubusercontent.com/16012374/151792323-a7f78a3e-355a-458c-97a9-4ce96ae5f247.png)

**Reply by Finii** [reply]
Just ran a font comparison in fontforge, ... the block building stuff differs; then letters with a dot, the dot has been moved in most; subscript numbers differ, the only **real** difference I can see is the small `t`:

![image](https://user-images.githubusercontent.com/16012374/151803010-acc7eb2a-2bd7-49e9-bc13-76ac074d715c.png)

_(green denotes outline change)_

**Comment by ggustafsson** [comment]
Oh, I just assumed it was two versions after reading the changelog part here [https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Inconsolata](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Inconsolata) and here [https://github.com/googlefonts/Inconsolata](https://github.com/googlefonts/Inconsolata).

I too noticed the updated ”t”. A little bit nicer. Other than that I assume I won’t notice anything. In fact I was even a bit surprised that I noticed anything at all :smiley:

If ”letters with a dot” means åäö etc then I might  notice it though as I’m Swedish…

**Reply by Finii** [reply]
No, the dieresis or umlaut stayed the same, its these:
![image](https://user-images.githubusercontent.com/16012374/151806964-a653d615-144e-4c12-a3fb-3908fa02b1e0.png)
The `dotbelowcomb` (which is glued to the basic glyph) has obviously changed position. As well as some other small 'will be combined with other  glyph' things.

**Comment by Finii** [comment]
See #775

**Comment by ggustafsson** [comment]
Thanks for the information and the pull request! 👌
